department-of-veterans-affairs/next-build
# feat: add va where link for facility location

# Description
Fix events "where link" to be a va link instead of a regular href

## Ticket
https://github.com/department-of-veterans-affairs/next-build/issues/670


## QA steps

- Run yarn test to verify i broke no tests
- View compiled html to verify compatibility. 

## Screenshots

Before:
![image](https://github.com/user-attachments/assets/3a6e215d-ecc2-4ba4-bc34-86c9c4dea2ce)

After:
![image](https://github.com/user-attachments/assets/87588638-72d5-493a-b679-81d069e5ea69)
